tests/Kernel/InstallationTest.php

```diff
@@ -20,7 +20,7 @@ class InstallationTest extends KernelTestBase {
    *
```

Comment:
Also add meaningful comment on this class.

Reply:
I have removed this test because we agreed that configuration could be done only manually.